Formula/q/qt.rb

```diff
@@ -141,18 +130,22 @@ class Qt < Formula
     sha256 "b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923"
   end
 
+  # Work around build failure with ninja >= 1.12.0.
+  # Issue ref: https://bugreports.qt.io/browse/QTBUG-124375
+  patch :DATA
```

Comment:
Hit another issue afterward for `fatal error: 'components/web_cache/public/mojom/web_cache.mojom.h' file not found` at `components/web_cache/browser/web_cache_manager.h`.

Not sure how to resolve this one, so going to use older `ninja` instead.